### PR TITLE
Cut release 0.2.0

### PR DESCRIPTION
**Pull request by matthiasr**
Bump version and add a changelog.

### CODE REVIEW

**Comment by matthiasr** [comment]
@juliusv

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## 0.2.0 / 2015-06-24
+
+* [IMPROVEMENT] Add Docker support.
+* [CHANGE] Switch logging to Prometheus' logging library.
+* [BUGFIX] Fix slave status parsing.
+* [BUGFIX] Fix truncated numbers.
+* [CHANGE] Reorganize metrics names and types.
+
+Breaking changes: Logging related flags have changed. Metric names have changed.
```
Comment (juliusv) [inline CHANGELOG.md]: "Logging-related"

I'd do "BREAKING CHANGES:" and move it to the very top.

Comment (matthiasr) [inline CHANGELOG.md]: done!

**Comment by juliusv** [comment]
:+1:

**Comment by matthiasr** [comment]
@juliusv could you also tag the release & attach tarballs?

**Comment by juliusv** [comment]
@matthiasr The `gh-release` tool (https://github.com/prometheus/utils#gh-release) only works properly on Mac so far. Could you run it? Then we also get cross-compiled binaries and everything.

I already created the tag and the GitHub release: https://github.com/prometheus/mysqld_exporter/releases

**Comment by matthiasr** [comment]
I don't have the necessary rights on this repo.

**Comment by juliusv** [comment]
@matthiasr Now you do!

**Pull request merged**